js/main.js

```diff
@@ -143,6 +143,59 @@ $('body').on('keypress', 'input', function(event) {
   }
 });
 
+//keyboard shortucts
+window.keyShortcuts = {
+	discover: 		'd',
+	myPage: 		'h',
+	customizePage: 	'e',
+	create: 		'n',
+	purchases: 		't',
+	sales: 			's',
+	cases: 			'b',
+	settings: 		','
```

Comment:
"E" is already used for "Customize page".

I'm not sure if windows users use any shortcuts to access Settings in their OS. In that case a mac-friendly shortcut wouldn't do any harm to them, but would be benefit for mac users. What key do linux users use?